cloud/passportauth/example/UserApi.ts

```diff
@@ -0,0 +1,37 @@
+import * as Promise from 'bluebird';
+import {BaseUser} from '../src/index';
+import UserDataRepository from './UserDataRepository';
+import {User} from './UserSeedData';
+
+/**
+ * A sample implementation of a base user api
+ */
+export class UserApi implements BaseUser {
+  protected userDataRepo: UserDataRepository;
+  protected user: User|undefined;
+
+  constructor(protected readonly dataRepo: UserDataRepository) {
+    this.userDataRepo = dataRepo;
```

Comment:
This is redundant, when you add `protected|public` to a constructor param it already creates a member in the class, so here you have `this.userDataRepo === this.dataRepo`, can remove one of them.